### ISSUE TITLE
Is it possible to change the path/filename of output files?

### ISSUE THREAD

**Issue by JonathanBristow** [ember-engines/ember-engines#513]
`dist/engines-dist/engine-name/assets/engine-vendor-d41d8cd98f00b204e9800998ecf8427e.css:`

Is it possible to change `engines-dist` in the path and ` engine-*` in the filename?

**Comment by villander** [comment]
No! This issue is already open, I think you can boost it there. #512
